styles.css

```diff
@@ -0,0 +1,214 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background: #222;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  min-height: 100vh;
+
+  color: white;
+
+  font-family: "Poppins", sans-serif;
+}
+
+div.container {
+  position: relative;
+}
+
+div.card {
+  position: relative;
+  height: 490px;
+  width: 350px;
+
+  overflow: hidden;
+
+  border: 3px solid white;
+  border-radius: 30px;
+
+  background-color: #131313;
+
+  box-shadow: 10px 10px 40px rgba(0, 0, 0, 0.9);
+}
+
+div.imgbox {
+  width: 350px;
+
+  padding-top: 5em;
+  /* border: 1px solid white; */
+
+  position: absolute;
+
+  z-index: 100;
```

Comment:
Why 100 not 1000?

Reply:
Because 100 was already the highest number. I don't use much big numbers until not needed, so It won't confuse me. If I use z-index feature multiple times.